dev/bots/entry.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# the bot entry-point - delegate to the correct script
+
+if [ "$BUILD_DOCS" = "true" ]; then
+  # generate docs
+  dev/bots/docs.sh
```

Comment:
we need to build the docs on every PR, otherwise once we start failing docs when there's bad references, etc, we won't catch them.